src/Unleash/DefaultUnleash.cs

```diff
@@ -57,6 +57,17 @@ public DefaultUnleash(UnleashSettings settings, params IStrategy[] strategies)
         ///// <param name="overrideDefaultStrategies">When true, it overrides the default strategies.</param>
         ///// <param name="strategies">Custom strategies.</param>
         public DefaultUnleash(UnleashSettings settings, bool overrideDefaultStrategies, params IStrategy[] strategies)
+            : this(settings, overrideDefaultStrategies, strategies, yggdrasilStrategies: null)
+        { }
+
+        ///// <summary>
+        ///// Initializes a new instance of Unleash client.
+        ///// </summary>
+        ///// <param name="config">Unleash settings</param>
+        ///// <param name="overrideDefaultStrategies">When true, it overrides the default strategies.</param>
+        ///// <param name="strategies">Custom strategies.</param>
+        ///// <param name="yggdrasilStrategies">Yggdrasil custom strategies.</param>
+        public DefaultUnleash(UnleashSettings settings, bool overrideDefaultStrategies, IStrategy[] strategies, params Yggdrasil.IStrategy[] yggdrasilStrategies)
```

Comment:
Decided on adding an overload for Yggdrasil strategies. They share name and some of the APIs are very much alike, but I don't like introducing inheritance between these two as that makes a mess of things. So for now Custom strategies for yggdrasil is a consciously made opt-in and shouldn't break things for existing users